monster_waves/RandomGenerator.hpp

```diff
@@ -0,0 +1,14 @@
+#include <ctime>
+#include <cstdlib>
+#include <algorithm>
+
+class RandomNumber
+{
+public:
+    RandomNumber(unsigned int minNumber, unsigned int maxNumber);
+
+    int getNumber();
+private:
+    unsigned int number;
+    unsigned int min, max;
```

Comment:
@JakubMastalski kompletnie niepotrzebne pola klasy. Do usunięcia. To samo tyczy się importów. Jeszcze raz ponawiam i zachęcam do zapoznania się z nowym sposobem generowania w C++:
https://en.cppreference.com/w/cpp/numeric/random

Możesz wybrać dowolny generator, ale zalecam Mersena